content/reference/endpoints/payment_gateways.md

```diff
@@ -3,32 +3,58 @@
 credentials                    | [Object](/reference/enums#formats) | Read       | Credentials used for payment gateway authentication.
 supported_cardtypes            | Array   | Read       | Array of String values containing supported cardtypes for selected gateway.
 supported_currencies           | Array   | Read       | Array of string values containing supported currencies for selected gateway.
-tokenize_credit_card_details   | Boolean | Read       | Credit Card details will be tokenized if set to true.
+tokenize_credit_card_details   | Boolean | Read       | Credit Card details will be tokenized if set to true before card's authorization.
 host_booking_pay_wallet_id     | String  | Read       | Wallet ID for BookingPay host.
 ---------------------|---------|------------|------------
 created_at           | [Time](/reference/enums#formats) | Read       | Payment Gateway's creation time.
 updated_at           | [Time](/reference/enums#formats) | Read       | Payment Gateway's creation time.
+{: class="table table-bordered"}
+  </div>
+  <div class="tab-pane" id="preferences_payments_write" markdown="1">
+Name                           | Type    | Read/Write | Description | Constraints 
+-------------------------------|---------|------------|-------------|
+account                        | Integer | Read       | Account's ID related to the Payment Gateway |
+-------------------------------|---------|------------|-------------|
+id                             | BigInteger | Read    | Uniq identifier. |
+name                           | String  | Read/Write       | Payment Gateway's name serving as an additional identifier. | **Required**, must not contain non-printing characters
+test                           | Boolean | Read/Write       | Submitted requests will not be charged if set to true (test mode/sandbox will be used). |
+gateway_name                   | String  | Read/Write       | Payment Gateway's name, available values are described in: [enums section](/reference/enums#payment-gateway-names) | **Required**, must be one of the available gateway names 
+credentials                    | [Object](/reference/enums#formats) | Read/Write | Credentials used for payment gateway authentication. | **Required**, the schema depends on `gateway_name`
+supported_cardtypes            | Array   | Read/Write       | Array of String values containing supported cardtypes for selected gateway, available values are described in: [enums section](/reference/enums#payment-gateway-cardtypes) | **Required**, must contain valid cardtypes supported by a given payment gateway
+supported_currencies           | Array   | Read/Write       | Array of string values containing supported currencies for selected gateway. | **Required**, must contain valid currencies
+tokenize_credit_card_details   | Boolean | Read/Write       | Credit Card details will be tokenized if set to true before card's authorization. |
```

Comment:
It might be worth mentioning that 3DS is not going to work for gateways with this value set to false and we override it to `true` for Stripe, Ogone, BookingPay in our payments processing flow http://developers.bookingsync.com/guides/secure-payments-by-bookingsync/#executing-payment

Reply:
@adamgrad The same applies for other cases so I wouldn't mention this flow explicitly as it could be misleading

Reply:
@Azdaroth not sure I understand, what exactly would be misleading?

Reply:
@adamgrad We also override it for B-com payments, Payment Requests etc., so if we mention that one, we would also need to mention other flows which is going a bit too bit, so I went in the end with a generic statement about payment flows.

Reply:
@Azdaroth why mentioning other flows? I think is is just enough to say that regardless of the value of this field, we'll always consider it `true` for gateways supporting 3DS and that's it

Reply:
@adamgrad But it's already there like that - I didn't mention any other flow, it's just the fact now that it's set to true for 3DS flows.